website/blog/2018-11-21-upgrade-node-version-circleci.md

```diff
@@ -0,0 +1,36 @@
+---
+title:     "How to upgrade node on CircleCI machine executor"
+author:    Manuel Mazzuola
+authorURL: https://keybase.io/originof
+---
+
+CircleCI offers 3 different [executor types](https://circleci.com/docs/2.0/executor-types/), machine, Docker and MacOS. On Linux the best choice would be the docker executor but sometimes you need full control over job environment so you must opt for the machine executor.
+
+The default machine executor image includes Docker, docker-compose, along with common language tools like nvm for node.  
+The issue we have encountered at ClouDesire is that if you need a different version of node you can't just install it via apt because the workspace uses nvm to manage node versions. And you can't just use nvm because on every run CircleCI resets the nvm configuration.
```

Comment:
```suggestion
The issue we have encountered at Cloudesire is that if you need a different version of node you can't just install it via apt because the workspace uses nvm to manage node versions. And you can't just use nvm because on every run CircleCI resets the nvm configuration.
```